lib/check_game_result.rb

```diff
@@ -0,0 +1,42 @@
+require_relative './win_vars'
+class CheckGameResult
+  def initialize(got_board)
+    @board = got_board
+  end
```

Comment:
Add `attr_readers/attr_writers/attr_accessors` for the above class instance variables.